CONTRIBUTING.md

```diff
@@ -1,37 +1,26 @@
 # Contributing to the OCM SDK
 
-### Validating model updates
```

Comment:
I see this title is removed, and moved into the other segment. I think it's deserve a segment on it's own. WDYT?